plugin/src/main/java/org/screamingsandals/bedwars/config/Configurator.java

```diff
@@ -148,15 +148,24 @@ public void createFiles() {
         checkOrSetConfig(modify, "farmBlocks.blocks", new ArrayList<>());
 
         checkOrSetConfig(modify, "scoreboard.enable", true);
-        checkOrSetConfig(modify, "scoreboard.title", "§a%game%§r - %time%");
-        checkOrSetConfig(modify, "scoreboard.bedLost", "§c\u2718");
-        checkOrSetConfig(modify, "scoreboard.bedExists", "§a\u2714");
+        checkOrSetConfig(modify, "scoreboard.title", "�a%game%�r - %time%");
+        checkOrSetConfig(modify, "scoreboard.bedLost", "�c\u2718");
+        checkOrSetConfig(modify, "scoreboard.bedExists", "�a\u2714");
         checkOrSetConfig(modify, "scoreboard.teamTitle", "%bed%%color%%team%");
 
         checkOrSetConfig(modify, "title.fadeIn", 0);
         checkOrSetConfig(modify, "title.stay", 20);
         checkOrSetConfig(modify, "title.fadeOut", 0);
-
+        
+        checkOrSetConfig(modify, "shop.rows", 6);
```

Comment:
default shop.rows for bedwars is four (it doesn't mean the actual inventory size but the part filled by items from shop.yml so without header and footer), render-actual-rows is 6 that's correct